alicloud/service_alicloud_sts.go

```diff
@@ -0,0 +1,23 @@
+package alicloud
+
+import (
+	"github.com/denverdino/aliyungo/sts"
+)
+
+func (client *AliyunClient) GetCallerIdentity() (*sts.GetCallerIdentityResponse, error) {
```

Comment:
I think this method is put into file service_alicloud_common is ok and there is no need to use a new file.

Reply:
Ok I can change it. I just wanted to avoid having too long files.